server/src/domain/handler.rs

```diff
@@ -44,7 +45,7 @@ pub struct CreateUserRequest {
     // Same fields as User, but no creation_date, and with password.
     pub user_id: UserId,
     pub email: String,
-    pub display_name: Option<String>,
```

Comment:
We want to keep the change inside this interface: it should only be the communication with the DB that changes. We still get a `Option<String>`, but then we convert it into a `DisplayName`